R/call_user_fun.R

```diff
@@ -24,6 +26,16 @@
 #'   weight = "hallo"
 #' )))
 call_user_fun <- function(call) {
+  deprecate_inform(
+    when = "1.3.0",
+    what = "call_user_fun()",
+    details = c(
+      "`call_user_fun()` is no longer in use; ",
+      "The original code for this function is here: ",
+      "https://github.com/pharmaverse/admiral/",
+      "blob/a49e84143379f921ef4e10ab546a09648d76f4ec/R/call_user_fun.R#L26-L39"
```

Comment:
It is easier to follow the link if it is in one line. You could shorten it by using a tag, e.g.,
```suggestion
      "https://github.com/pharmaverse/admiral/blob/v1.2.0/R/call_user_fun.R#L26-L39"
```